Formula/rclone.rb

```diff
@@ -1,10 +1,8 @@
-require "language/go"
-
 class Rclone < Formula
   desc "Rsync for cloud storage"
   homepage "https://rclone.org/"
-  url "https://github.com/ncw/rclone/archive/v1.36.tar.gz"
-  sha256 "a573b70e3aeb355b943dddd6ae9375386fc21bf12dfba601d8d7280f97c4c884"
+  url "https://github.com/ncw/rclone/archive/v1.37.tar.gz"
+  sha256 "a74e284d2368f6fb8e4ac654a31b9b1328ac6078acd3446c9a892cd4bcbe8660"
 
```

Comment:
Mind adding

```
head "https://github.com/ncw/rclone.git"
```

I need the head version, which could be squeezed into this PR.